src/java/org/apache/cassandra/guardrails/Guardrails.java

```diff
@@ -89,6 +89,11 @@
                                                                                                     String::toLowerCase,
                                                                                                     "Table Properties");
 
+    public static final Guardrail.IgnoredValues<String> ignoredTableProperties = new Guardrail.IgnoredValues<>("ignored_table_properties",
```

Comment:
nit: we should import `IgnoredValues` similar other guardrails (e.g. `new DisallowedValues` vs `new Guardrail.IgnoredValues`).